### ISSUE TITLE
Comando de dicionário sem suporte a outras linguagens 

### ISSUE THREAD

**Issue by CanasDev** [LorittaBot/DreamLand#190]
**Qual é o problema?**
Olá, bom dia quando eu seto a linguagem da loritta para inglês, o comando de dicionário fica em português (não sei se isso é um bug ou se é mesmo assim)
**Como reproduzir o problema?**

1. Use {prefixo}setlang 
2. Sete a linguagem do servidor para inglês 
3. Use o comando {prefixo}dicio

**Considerações Finais**

Gatos são muito lindos.
![Screenshot_20210815_094103](https://user-images.githubusercontent.com/70149057/129472540-bb8753a1-12d3-44b9-b4cb-18a151834ba4.jpg)

**Comment by MrPowerGamerBR** [comment]
Sim, atualmente ela só pega do dicio.com.br, por isso é em português, mas está nos planos ter em outras linguagens! (Tanto que a versão em slash command, `/dictionary`, pergunta qual linguagem você deseja buscar, mas no momento só tem português)
